mlir/include/mlir/Dialect/EmitC/Transforms/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls -name EmitC)
+mlir_tablegen(Passes.capi.h.inc -gen-pass-capi-header --prefix EmitC)
+mlir_tablegen(Passes.capi.cpp.inc -gen-pass-capi-impl --prefix EmitC)
```

Comment:
It's building now, but it looks like the generated `Passes.capi.cpp.inc` implementation files are not actually being used.

From looking at how it's done for other dialects, it looks like you still need to create a .cpp file, and include it:

https://github.com/llvm/llvm-project/blob/57abd4e4abb705a453134051743542de5fd396bc/mlir/lib/CAPI/Dialect/GPUPasses.cpp#L1-L26

Reply:
(otherwise, using any of these CAPIs will result in linker errors)